src/audiomixerboard.cpp

```diff
@@ -291,36 +293,67 @@ void CChannelFader::SetDisplayPans ( const bool eNDP )
 
 void CChannelFader::SetupFaderTag ( const ESkillLevel eSkillLevel )
 {
-    // the group ID defines the border color
-    QString strBorderColor;
+    // Should never happen here
+    if ( iGroupID >= MAX_NUM_FADER_GROUPS )
+    {
+        SetGroupID ( INVALID_INDEX );
+    }
+
+    // the group ID defines the border color and style
+    QString strBorderColor = "black";
+    QString strBorderStyle = "solid";
 
-    switch ( iGroupID )
+    if ( iGroupID != INVALID_INDEX )
     {
-    case 0:
-        strBorderColor = "red";
-        break;
+        switch ( iGroupID % 4 )
+        {
+        case 0:
+            strBorderColor = "#C43AC5";
+            break;
 
-    case 1:
-        strBorderColor = "blue";
-        break;
+        case 1:
+            strBorderColor = "#2B93D4";
+            break;
 
-    case 2:
-        strBorderColor = "green";
-        break;
+        case 2:
+            strBorderColor = "#3BC53A";
+            break;
 
-    case 3:
-        strBorderColor = "yellow";
-        break;
+        case 3:
+            strBorderColor = "#D46C2B";
+            break;
 
-    default:
-        strBorderColor = "black";
-        break;
+        default:
+            break;
+        }
+
+        switch ( iGroupID / 4 )
+        {
+        case 0:
+            strBorderStyle = "solid";
+            break;
+
+        case 1:
+            strBorderStyle = "dashed";
+            break;
+
+        case 2:
```

Comment:
2 and 3 currently not used.  These allow for up to 16 groups.  Beyond that, good contrasting styling becomes more problematic.  And probably harder to cope with mentally.  "Five plus or minus two" applies to many things - so eight groups should be more than enough.